python/svix/exceptions.py

```diff
@@ -1,25 +1,11 @@
-from .openapi_client.exceptions import (
-    ApiAttributeError,
-    ApiException,
-    ApiKeyError,
-    ApiTypeError,
-    ApiValueError,
-    ForbiddenException,
-    NotFoundException,
-    ServiceException,
-    UnauthorizedException,
-)
+from .internal.openapi_client.models.http_error import HttpError
+from .internal.openapi_client.models.http_validation_error import HTTPValidationError
+from .internal.openapi_client.models.validation_error import ValidationError
 from .webhooks import WebhookVerificationError
 
 __all__ = [
-    "ApiAttributeError",
-    "ApiException",
-    "ApiKeyError",
-    "ApiTypeError",
-    "ApiValueError",
-    "ForbiddenException",
-    "NotFoundException",
-    "ServiceException",
-    "UnauthorizedException",
```

Comment:
hmm this is a breaking change but maybe we're okay with it? otherwise we need to do more work to diferentiate the httperrors that you raise